### ISSUE TITLE
Improve resistor-color: detect default value for argument and disapprove

### ISSUE THREAD

**Issue by SleeplessByte** [exercism/javascript-analyzer#73]
**Describe the improvement**:

When a solution has a default value for the `color` argument, the analyzer bails out. It should detect this and disapprove instead.

**Which exercise**

`resistor-color`

**Example code**

```javascript
export const colorCode = (color='black') => COLORS.indexOf(color);
export const COLORS = ["black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"];
 ```

**Comment by SleeplessByte** [comment]
Now returns:

```json
{
  "comments": [
    {
      "comment": "javascript.general.unexpected_parameter",
      "params": {
        "type": "AssignmentPattern"
      },
      "type": "actionable"
    }
  ]
}
```

Optimal would be to detect the usage of a default value, but 🤷🏽
